stories/Core Components/ShareDialog/OnMount.svelte

```diff
@@ -0,0 +1,11 @@
+<script lang="ts">
```

Comment:
`OnMount` is outdated way to mount components. Better to use this approach https://github.com/santiment/san-webkit/blob/master/stories/Subscription/Payment%20Dialog/index.svelte#L8

Reply:
Fixed